tests/e2e/e2e_trafficsplit_same_sa_test.go

```diff
@@ -80,7 +80,8 @@ var _ = OSMDescribe("Test TrafficSplit where each backend shares the same Servic
 							Namespace:    serverNamespace,
 							ReplicaCount: int32(serverReplicaSet),
 							Image:        "simonkowallik/httpbin",
-							Ports:        []int{80},
+							Ports:        []int{14001},
+							Command:      []string{"gunicorn", "-b", "0.0.0.0:14001", "httpbin:app", "-k", "gevent"},
```

Comment:
Why is this additional command needed only for this test?

Reply:
The tests that use this image also need this change but @SanyaKochhar will implement those changes so that she can learn

Reply:
Since the same port is going to be used across all the tests, it makes sense to define the slices as pkg scoped variables that other tests can reference.